bc_events/utils.py

```diff
@@ -25,12 +28,58 @@ def build_topic_name(category, entity, action):
     return "{category}.{entity}{action}".format(category=category, entity=entity, action=action)
 
 
-def requests_session():
-    """For use on AWS APIs that periodically fail with 500 and suggest retrying"""
+class EventsApiRetryingWrapper(object):
+    def __init__(self, url, payload, headers={}, delay=0.45, max_delay=5, max_time=35):
```

Comment:
I think we should be a little more aggressive on our retry timing and shorten all of these values. We really don't want posting an event to introduce much latency, as it will almost always be a part of another API call.

From the events being sent so far, current usage is coming in < 100ms.

```suggestion
    def __init__(self, url, payload, headers={}, delay=0.1, max_delay=0.5, max_time=2):
```

Reply:
Sounds good! The suggested changes have been made.